packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -107,19 +108,29 @@ const PreviewTab = ({ input, sourceId, parentSelector }: PreviewTabProps) => {
 
 export default function NewCommentModal({
   onRequestClose,
-  editId,
   onComment,
   onInputChange,
-  replyTo,
-  editContent,
   parentSelector,
+  parentComment,
+  post,
   ...props
 }: NewCommentModalProps): ReactElement {
+  const { handle, commentId, editId, editContent, replyTo, authorId } =
+    parentComment;
   const [input, setInput] = useState<string>(editContent || replyTo);
   const [sendingComment, setSendingComment] = useState<boolean>(false);
   const [errorMessage, setErrorMessage] = useState<string>(null);
   const { showPrompt } = usePrompt();
   const { requestMethod } = useRequestProtocol();
+  const { data: member, isFetched: isMembershipFetched } = useQuery(
+    generateQueryKey(RequestKey.PostComments, null, authorId),
```

Comment:
@rebelchris we should add `post.source.id` also to the query key.

Data would potentially be invalid if looking at the comment from the same author on different source (squad) because it would already be populated from previous comment membership.

Reply:
Ah good point indeed! I'll add that :)